crates/oxide/src/parser.rs

```diff
@@ -303,6 +307,14 @@ impl<'a> Extractor<'a> {
             }
         }
 
+        // CSS variables must be preceded by `var(` to be considered a valid CSS variable candidate
+        if candidate.starts_with(b"--") {
+            match input.get(start_idx - 4..start_idx) {
+                Some(b"var(") => return ValidationResult::Valid,
```

Comment:
Also `getPropertyValue`, `theme(`, and `--theme(` at the very least. We probably should just track everything that starts with `--` so that  this also works:

```js
let var = '--my-var';
let style = bla.getPropertyValue(var);
```